R/data_copy.R

```diff
@@ -205,7 +205,7 @@ spark_data_copy <- function(
     stop("The repartition parameter must be an integer")
   }
 
-  if (!spark_connection_is_local(sc) && identical(serializer, "csv_file")) {
+  if ((!spark_connection_is_local(sc) || sc$method == "databricks-connect") && identical(serializer, "csv_file")) {
```

Comment:
Is it safer to update `spark_conection_is_local()` to return false when we are running on Databricks Connect? I wonder what other cases have different behavior. Would be good to check.